backends/cuda-gen/ceed-cuda-gen-operator-build.cpp

```diff
@@ -215,13 +214,15 @@ static int CeedOperatorBuildKernelRestriction_Cuda_gen(std::ostringstream &code,
   CeedEvalMode              eval_mode  = CEED_EVAL_NONE;
   CeedInt                   elem_size = 0, num_comp = 0, P_1d = 0;
   CeedSize                  l_size;
+  CeedRestrictionType       rstr_type = CEED_RESTRICTION_STANDARD;
   CeedElemRestriction_Cuda *rstr_data;
   CeedElemRestriction       elem_rstr;
   CeedBasis                 basis;
 
   // Get field data
   CeedCallBackend(CeedOperatorFieldGetElemRestriction(op_field, &elem_rstr));
   if (elem_rstr != CEED_ELEMRESTRICTION_NONE) {
+    CeedCallBackend(CeedElemRestrictionGetType(elem_rstr, &rstr_type));
```

Comment:
Side note: `CeedRestrictionType` isn't documented currently. Honestly not sure what it is or does.

Reply:
I should go ahead and toss a docstring on it. Short version is that its an enum of the different types of restrictions: standard, strided, oriented, curl oriented, and at points.